### PR TITLE
Hacktoberfest-2022 Update blog page

### PR DESCRIPTION
**Pull request by Anuran12**
## Related Issue

- Information about the related issue
Close: #196 

### Describe the changes you've made

I have change the Blog page... I made this change according to the contributors page theme 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?
I created a local hosting to test the changes . I tested for various screen ratio to check the responsive

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | image

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/210"></a>

### CODE REVIEW

**Review by krshkodes — CHANGES_REQUESTED** [review]
Can you let the breakpoints and variable names remain the same? They might cause issues elsewhere otherwise

**Comment by Anuran12** [comment]
> Can you let the breakpoints and variable names remain the same? They might cause issues elsewhere otherwise

well the breakpoints are already using by default .
image

the sm,md,lg.xl are default screen size for tailwind.. 

image

And those are already using in "Navbar.js" 

I just added the 'xs' additionally . because tailwind does not provide us 360px screen width by default .

and without that the blog cards are looking bad in phone view. so i just added 'xs': '360px',

I guess this will not harm any other code.

**Review by krshkodes — CHANGES_REQUESTED** [review]
Tailwind does provide xs, but it isn't the size you want - I would suggest that you use a grid layout for responsiveness and not touch the tailwind config more than needed

**Comment by Anuran12** [comment]
> Tailwind does provide xs, but it isn't the size you want - I would suggest that you use a grid layout for responsiveness and not touch the tailwind config more than needed

@krshkodes I Made changes according to your advice.. Please check it : )

File: src/pages/blog.js [hunk 1]
```diff
@@ -1,34 +1,69 @@
...
+  const data = await res.json();
+
+  return {
+    props: {
+      data,
+    },
+  };
+};
+
+const blogs = ({ data }) => {
```
Comment (mkubdev) [inline src/pages/blog.js:15]:
```suggestion
const Blog = ({ data }) => {
```


Hi, [Pages]https://nextjs.org/docs/basic-features/pages) should be exported with Capitalized letter! :D

File: src/pages/blog.js [hunk 2]
```diff
@@ -1,34 +1,69 @@
...
-  });
-  const articles = await res.json();
-  return {
-    props: {
-      articles,
-    },
-    revalidate: 10,
-  };
-}
+export default blogs;
```
Comment (mkubdev) [inline src/pages/blog.js:69]:
```suggestion
export default Blog;
```

File: src/pages/blog.js [hunk 3]
```diff
@@ -1,34 +1,69 @@
...
-          <h1 className='font-bold text-5xl antialiased'>Check out our blog posts</h1>
-          <div className='mt-6 text-xl font-light text-true-gray-500 antialiased'>
-            Read our articles from <span className='font-mono'>Dev.to</span>
+      <section className='py-8'>
+
+        <div className='container max-w-5xl mx-auto'>
+          <h1 className='w-full my-2 text-4xl font-bold leading-tight text-center text-white'>
+            Blogs
+          </h1>
+          <div className='w-full mb-4'>
```
Comment (mkubdev) [inline src/pages/blog.js:27]:
```suggestion
          <div className="w-full mb-4">
```

Hi please, kindly suggest you to update the className with double quote: ` " `

**Review by mkubdev — CHANGES_REQUESTED** [review]
Hi 🎉
This look good to me! But it requires few changes.

If you use VSCode, i suggest you to use the [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) plugin to format your files. So that single quotes are auto formated into double quotes. 🥳

Good job ⚡🎉

**Comment by Anuran12** [comment]
@mkubdev Done...

**Review by mkubdev — APPROVED** [review]
This look good to me!

**Review by vinzvinci — APPROVED** [review]
LGTM!

**Comment by Anuran12** [comment]
@mkubdev you can marge this.. because this page is no need any update or improvement now.

**Comment by mkubdev** [comment]
Hello @Anuran12 !

Yes, let's merge your changes. Thanks a lot for your valuable contribution and your implication. Very good work! ⚡

Please join us on our [Discord](https://discord.gg/fah5cTBN), we are looking for maintainers/collaborators!

**Comment by Anuran12** [comment]
Hey.. @mkubdev 

I have joined Discord..

I would like to be a maintainers/collaborators.. But I am a newbie so please guide me. This opportunity will give me more exposer.

Thanks  : )

**Comment by mkubdev** [comment]
What is your Discord username?
Cool if you already joined. Don't worry, we all started learning like you!

**Comment by Anuran12** [comment]
Anuran#2573

**Pull request merged**